src/component/AddItem.js

```diff
@@ -32,13 +52,29 @@ const AddItem = ({ firestore }) => {
   };
 
   const handleOptionChange = event => {
-    setFrequency(event.target.value);
+    setNumberOfDays(event.target.value);
+  };
+
+  // function triggered at handleSubmit -
+  // normalizes item name so that it has all lowercase
+  // letters and no special characters (spaces ok)
+  const normalizeName = name => {
```

Comment:
This is an interesting approach! I can see a couple of limitations to it, though:

1. It doesn't retain capitalization nor punctuation (in other words, can we compare the strings _as if_ those things were normalized, but still store the original string?)
2. It strips accented characters (no espárragos for you!)
3. I was able to submit an item with a trailing space (which, technically doesn't _not_ fulfill the AC, so I won't count it against you!)

I do like the simplicity of your approach! I also **love** that it's pulled out into its own function. Can we try to make the solution satisfy #s 1 and 2 above?